hack/ci/delete-cluster.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
This isn't used in CI it seems, but could be useful when running locally I guess 👍🏼